cmd/telemetrygen/internal/metrics/worker.go

```diff
@@ -28,7 +29,13 @@ type worker struct {
 
 func (w worker) simulateMetrics(res *resource.Resource, exporter sdkmetric.Exporter) {
 	limiter := rate.NewLimiter(w.limitPerSecond, 1)
+
 	var i int64
+	value := 24.42
+	attrs := attribute.NewSet(attribute.KeyValue{
```

Comment:
Let's also move the possibility of adding attributes to a separate PR

Reply:
Removed attributes. Will create a separate PR

Reply:
Thanks! Have a look at #26506, it may do everything you wanted